quotes/tests.py

```diff
@@ -383,3 +382,38 @@ def test_delete_quote__bad_http_method__fails_with_405(self):
         response = self.client.put(reverse(quotes_delete, kwargs={'quote_id': 2}))
         self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
 
+
+class QuoteApproveTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = User(
+            username='test',
+            email='test@example.com',
+            first_name='Alex',
+            last_name='Orvik'
+        )
+        cls.user.set_password('password')
+        cls.user.save()
+
+        cls.quote = Quote(
+            text='Some quote text',
+            quoter=cls.user,
+            id=124
+        )
+        cls.quote.save()
+
+    def setUp(self):
+        self.client.login(username='test', password='password')
+
+    def test_approving_unapproved_quote(self):
```

Comment:
Please see [here](https://github.com/KSG-IT/ksg-nett/blob/develop/CONTRIBUTING.md#automated-testing) for test naming.